src/applications/vaos/utils/eligibility.js

```diff
@@ -99,9 +100,9 @@ export async function getEligibilityData(
   const eligibility = {
     ...results,
     hasMatchingClinics: !!results.clinics?.length,
-    directSupported: location.legacyVAR.directSchedulingSupported,
+    directSupported: useVSP || location.legacyVAR.directSchedulingSupported,
```

Comment:
Does this imply that direct scheduling is always supported with VSP?

Reply:
Yes, for now. I figured it was preferable to be able to test the VSP flow before all the eligibility checks get implemented in VSP, and I didn't want to add the `legacyVAR` object to data that actually came from VSP.